tests/results/DiffTest.py

```diff
@@ -37,13 +37,15 @@ def test_delete_line(self):
         additions, deletions = self.uut.stats()
         self.assertEqual(deletions, 1)
         self.assertRaises(ValueError, self.uut.delete_line, 0)
+        self.assertRaises(ValueError, self.uut.delete_line, 10)
 
     def test_delete_lines(self):
-        self.uut.delete_lines(1, 10)
-        self.uut.delete_lines(10, 20)
+        self.uut.delete_lines(1, 2)
+        self.uut.delete_lines(2, 3)
         additions, deletions = self.uut.stats()
-        self.assertEqual(deletions, 20)
-        self.assertRaises(ValueError, self.uut.delete_lines, 0, 10)
+        self.assertEqual(deletions, 3)
```

Comment:
10,20 is out of range for diff. That's why also I have added tests to handle cases like that. https://github.com/coala/coala/pull/3817/files#diff-8ab4f61ecc762c2ad5312562c454b2aaR48